lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -87,6 +96,24 @@ def script
             fi
           }
 
+          # Strip invalid architectures
+          strip_invalid_archs() {
```

Comment:
No, I think they are just used on link time. I didn't remembered that we filter them out on rsync already. :white_check_mark: